src/main/java/com/agilethought/internship/sso/controller/GlobalExceptionHandler.java

```diff
@@ -1,44 +0,0 @@
-package com.agilethought.internship.sso.controller;
```

Comment:
You should not delete this class, because belongs to previous implementation and we want preserve it